dynawo/sources/Models/CPP/Components/ModelLoadRestorativeWithLimits/DYNModelLoadRestorativeWithLimits.cpp

```diff
@@ -135,7 +135,7 @@ namespace DYN {
     variables.push_back(VariableNativeFactory::createState("Ii_value", FLOW));
     variables.push_back(VariableNativeFactory::createState("PPu_value", CONTINUOUS));
     variables.push_back(VariableNativeFactory::createState("QPu_value", CONTINUOUS));
-    variables.push_back(VariableNativeFactory::createCalculated("state_value", CONTINUOUS));
+    variables.push_back(VariableNativeFactory::createCalculated("state_value", INTEGER));
```

Comment:
shouldn't this be done in another PR as unrelated?

Reply:
wrong manipulation with git, sorry, solved